thentos-core/src/Thentos/Backend/Api/Simple.hs

```diff
@@ -161,6 +163,11 @@ thentosServiceSession =
   :<|> endServiceSession
 
 
+-- * send email
+
+type SendEmail =
+    ReqBody '[JSON] SendEmailRequest :> Post '[JSON] ()
+
 -- * servant docs
```

Comment:
There are some layout style rules for haddock sections:

https://github.com/liqd/thentos/blob/master/docs/styleguide.md#layout

(There is also the rule that if you don't like anything in the style guide, you can write a PR.  And regarding haddock section headings, I'm not sure they even make sense in the presence of explicit export list.  I think in that case they should go to the export list, so this rule needs some work.  Anyway, this is what it is now...)